code/game/objects/items/devices/ocd.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/devices/ocd_device
+    //HOW HAVE I SURVIVED THIS LONG WITHOUT BRAINCELLS?!
+	name = "Occupational Corruption Device"
+	desc = "When you need to make the lives of new-hires that much more confusing, think OCD."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-white"
+
+/obj/item/devices/ocd_device/attack_self(mob/user)
+    var/datum/round_event/bureaucratic_error/event = new(queue_event = FALSE) //God I'm fucking stupid
+    event.start() //I'm REALLY fucking stupid

```

Comment:
No need to self deprecate in comments, the pr is working, you're not dumb

Reply:
Lemon, the self-deprecation is me forgetting dumb shit or being too dumb to be able to dissect it myself